README.md

```diff
@@ -0,0 +1,30 @@
+# ansible-loki-callback: An Ansible callback plugin that logs to a loki instance
+
+## Requirements
+
+* Current Python3 implementation
```

Comment:
```suggestion
* Python3
```
...why "current"? What does "current" mean? Special version? If so, it should be described more explicit